internal/model/date.go

```diff
@@ -0,0 +1,82 @@
+package model
+
+import (
+	"strings"
+	"time"
+)
+
+type Date struct {
+	Time time.Time
```

Comment:
I think this should be private, so rename to `time` or `t`

Reply:
Need this in tests not changed the capital letter